catalog/resource_metastore_test.go

```diff
@@ -154,11 +161,9 @@ func TestForceDeleteMetastore(t *testing.T) {
 	qa.ResourceFixture{
 		Fixtures: []qa.HTTPFixture{
 			{
-				Method:   "DELETE",
-				Resource: "/api/2.1/unity-catalog/metastores/abc",
-				ExpectedRequest: map[string]bool{
-					"force": true,
-				},
+				Method:          "DELETE",
+				Resource:        "/api/2.1/unity-catalog/metastores/abc",
+				ExpectedRequest: map[string]interface{}{"-": "abc", "force": true},
```

Comment:
Yuck! The next release of the Go SDK will fix this request to not include the `-` key.